uui/components/overlays/NotificationCard.tsx

```diff
@@ -11,19 +11,26 @@ import { ReactComponent as HintIcon } from '../../icons/notification-help-fill-2
 import { ReactComponent as CrossIcon } from '../../icons/snackbar/cross.svg';
 import css from './NotificationCard.module.scss';
 
-interface NotificationAction extends IHasRawProps<React.ButtonHTMLAttributes<HTMLButtonElement>> {
+export interface NotificationAction extends IHasRawProps<React.ButtonHTMLAttributes<HTMLButtonElement>> {
+    /** NotificationAction name */
     name: string;
+    /** Notification action */
     action: () => void;
 }
 
-export interface DefaultNotificationProps extends INotification, IHasChildren, IHasCX, IHasRawProps<React.HTMLAttributes<HTMLDivElement>> {
+export type NotificationCardCoreProps = INotification & IHasChildren & IHasCX & IHasRawProps<React.HTMLAttributes<HTMLDivElement>> & {
+    /** Array of Notification actions */
```

Comment:
Add - 'If provided will be displayed as LinkButtons in the end of notification'

Reply:
fixed